python_scripts/functions.py

```diff
@@ -16,23 +16,17 @@
 s3_client = boto3.client("s3")
 
 
-def load_and_validate_config():
+def load_and_validate_config(path):
```

Comment:
Code is good, however I've also made changes to this as well. Might just need to spend Monday merging stuff in together as we are going to get a few messy conflicts.